README.md

```diff
@@ -1,32 +1,22 @@
 Caddy Module: http.handlers.cache
 ================================
 
-**⚠️ Work-in-progress**
-
-This is a distributed HTTP cache module for Caddy.
+This is a distributed HTTP cache module for Caddy based on [Souin](https://github.com/darkweak/souin) cache.
 
 ## Features
 
-* Supports most HTTP cache headers defined in [RFC 7234](https://httpwg.org/specs/rfc7234.html) (see the TODO section for limitations)
-* Sets [the `Cache-Status` HTTP Response Header](https://httpwg.org/http-extensions/draft-ietf-httpbis-cache-header.html)
+ * Supports most HTTP cache headers defined in [RFC 7234](https://httpwg.org/specs/rfc7234.html) (see the TODO section for limitations)
+ * Sets [the `Cache-Status` HTTP Response Header](https://httpwg.org/http-extensions/draft-ietf-httpbis-cache-header.html)
+ * REST API to purge the cache and list stored resources.
+ * Builtin support for distributed cache.
+
+
+## Example Configurations
 
-## Example Configuration
+You can find an example for the [Caddyfile](Caddyfile) or the [JSON file](configuration.json).  
+See the [Souin](https://github.com/darkweak/souin) configuration for the full configuration, and its associated [Caddyfile](https://github.com/darkweak/souin/blob/master/plugins/caddy/Caddyfile)  
```

Comment:
Is it intended to remain compatible via config like this, or does the user need to switch to the Souin plugin to leverage the features being discussed as not supported in `cache-handler`?

Reply:
I think I should provide a minimal configuration for Caddy in the main Caddyfile (the one on at the root project directory).